pdns/recursordist/ecs.cc

```diff
@@ -0,0 +1,38 @@
+#include "syncres.hh"
```

Comment:
So far recursordist has not seen actual .cc files, just symlinks one dir up. However, I feel that we should be moving in this direction anyway (i.e. unshared files -should- live in recursordist/), so unless @pieterlexis disagrees, I suggest keeping it like you did.